packages/js/src/ui/components/PopoverWithBell.tsx

```diff
@@ -0,0 +1,20 @@
+import { createSignal, useContext } from 'solid-js';
+import { Bell, Popover } from '.';
+
+export const PopoverWithBell = (props: any) => {
+  const [bellRef, setBellRef] = createSignal<HTMLDivElement | null>(null);
+  const [isOpen, setIsOpen] = createSignal(false);
+
+  const handlePopover = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  return (
+    <>
+      <div class="nt-relative" ref={setBellRef} onClick={handlePopover}>
+        <Bell />
+        <Popover isOpen={isOpen()} targetRef={bellRef()} placement="bottom"></Popover>
```

Comment:
We need to make sure this is accessible as well. It needs to close via `esc` and by clicking outside the popover. However, the logic should be in the popover implementation itself.